lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -782,6 +800,191 @@ setNonEnumerableReadOnly( REPL.prototype, 'clearUserDocs', function clearUserDoc
 	return this;
 });
 
+/**
+* Gets all available themes in the syntax-highlighter.
+*
+* @name themes
+* @memberof REPL.prototype
+* @type {Function}
+* @returns {Array} array of all theme names
+*
+* @example
+* var debug = require( '@stdlib/streams/node/debug-sink' );
+*
+* // Create a new REPL:
+* var repl = new REPL({
+*     'output': debug()
+* });
+*
+* // ...
+*
+* // Fetch all available themes:
+* repl.themes();
+*
+* // ...
+*
+* // Close the REPL:
+* repl.close();
+*/
+setNonEnumerableReadOnly( REPL.prototype, 'themes', function themes() {
+	return this._syntaxHighlighter.getThemes();
+});
+
+/**
+* Gets a theme's color palette from the syntax-highlighter.
+*
+* @name getTheme
+* @memberof REPL.prototype
+* @type {Function}
+* @param {string} theme - theme name
+* @throws {TypeError} argument must be a string
```

Comment:
```suggestion
* @throws {TypeError} must provide a string
```

This matches the convention found elsewhere in the project.